Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -98,11 +98,11 @@ void VideoConfigDiag::Event_Close(wxCloseEvent& ev)
 }
 
 #if defined(_WIN32)
-static wxString backend_desc = wxTRANSLATE("Selects what graphics API to use internally.\nThe software renderer is only used for debugging, so you'll want to use either Direct3D or OpenGL. Different games will behave differently on each backend, so for best emulation experience it's recommended to try both and chose the one that fits your requirements best.\nNote that the Direct3D backend is not available on old Windows versions.\n\nIf unsure, use OpenGL.");
+static wxString backend_desc = wxTRANSLATE("Selects what graphics API to use internally.\nThe software renderer is extremely slow and only useful for debugging, so you'll want to use either Direct3D or OpenGL. Different games will behave differently on each backend, so for the best emulation experience it's recommended to try both and choose the one that has less issues.\n\nIf unsure, use OpenGL.");
```

Comment:
I don't understand. The page you linked to says to use less with uncountable things and units of measurement and fewer everywhere else. This isn't uncountable nor is it a unit of measurement, so according to that page fewer should be used, but I'm fine with using less if you want.